File: universal7885-common/apps/CameraLightSensor/src/com/eurekateam/cameralightsensor/BootReceiver.java
package com.eurekateam.cameralightsensor;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import com.eurekateam.cameralightsensor.CameraLightSensorService;

public class BootReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if (intent.getAction().equalsIgnoreCase(Intent.ACTION_BOOT_COMPLETED)) {
            if (context != null) {
                final Intent i = new Intent(context, CameraLightSensorService.class);
                context.startForegroundService(i);
            }
        }
    }
}

File: universal7885-common/apps/CameraLightSensor/src/com/eurekateam/cameralightsensor/BootReceiver.kt
package com.eurekateam.cameralightsensor

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action.equals(Intent.ACTION_BOOT_COMPLETED, ignoreCase = true)) {
            val i = Intent(context, CameraLightSensorService::class.java)
            context.startForegroundService(i)
        }
    }
}